SimeonInstaller.ps1

```diff
@@ -2621,7 +2621,7 @@ CRLFOption=CRLFAlways
             # Add project collection build service to Project Collection Build Service Accounts
             Write-Information "Adding Project Collection Build Service to Project Collection Build Service Accounts ($Organization)"
             $buildServiceAccountsGroupDescriptor = ($groups |? principalName -eq "[$Organization]\Project Collection Build Service Accounts").descriptor
-            $projectCollectionBuildServiceDescriptor = ($users |? displayName -like  "Project Collection Build Service ($Organization)").descriptor
+            $projectCollectionBuildServiceDescriptor = ($users |? displayName -like  "Project Collection Build Service (*)").descriptor
```

Comment:
Line 2618 seems to do the same thing and is disregarding the final parenthesis
Should probably work anyway, but would be good to check